usage/src/main/java/org/killbill/billing/usage/api/svcs/DefaultInternalUserApi.java

```diff
@@ -54,6 +58,8 @@ public DefaultInternalUserApi(final RolledUpUsageDao rolledUpUsageDao,
     @Override
     public List<RawUsageRecord> getRawUsageForAccount(final LocalDate startDate, final LocalDate endDate, final InternalTenantContext internalTenantContext) {
 
+        log.info("GetRawUsageForAccount startDate='{}', endDate='{}'", startDate, endDate);
```

Comment:
`debug`?

Reply:
Was thinking of keeping this one as INFO on purpose (this is just a high level trace to see which calls we make per account-recordId